app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -807,7 +808,12 @@ private void populateResultsBox(IVScanResult ivScanResult) {
      *
      */
     private void adjustSeekbarForPokemon() {
-        expandedLevelSeekbar.setProgress(levelToSeekbarProgress(Data.trainerLevelToMaxPokeLevel(trainerLevel)));
+        expandedLevelSeekbarBackground.getThumb(0).setThumb(getResources().getDrawable(R.drawable
+                .orange_scrubber_control_disabled_holo));
+        expandedLevelSeekbarBackground.getThumb(0).setValue(levelToSeekbarProgress(trainerLevel + 1.5f));
```

Comment:
That's `Data.trainerLevelToMaxPokeLevel(trainerLevel)` not  `trainerLevel + 1.5f` (they're not always the same, the former has a few bugfixes).